Project.toml

```diff
@@ -7,9 +7,11 @@ version = "0.2.8"
 
 [deps]
 CategoricalArrays = "324d7699-5711-5eae-9e2f-1d82baa6b597"
+CompilerSupportLibraries_jll = "e66e0078-7015-5450-92f7-15fbd957f2ae"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+JLLWrappers = "692b3bcd-3c85-4b1f-b108-f13ce0eb3210"
```

Comment:
Why `JLLWrappers` was added as a dependency?  It doesn't seem to be used anywhere besides in this file: https://github.com/search?q=repo%3ANosferican%2FEconometrics.jl+JLLWrappers&type=code